.github/workflows/npm-release.yml

```
@@ -0,0 +1,58 @@
+      - name: Set up Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: "22"
+
+      - name: Install dependencies
+        working-directory: ./${{inputs.app-path}}
+        run: npm ci
+        
+      - name: Install dependencies1
+        working-directory: ./${{inputs.app-path}}
+        run: | 
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:6:9: Double quote to prevent globbing and word splitting

(shellcheck)